src/utils/setCookieExpire.ts

```diff
@@ -0,0 +1,5 @@
+// all cookie expires after 14days
+
+export const setCookieExpires = () => {
+    return new Date(Date.now() + 14 * 24 * 60 * 60 * 1000).toUTCString()
```

Comment:
can we move it as

const days = 24 * 60 * 60 * 1000
export const setCookieExpires = () => {
    return new Date(Date.now() + 14 * days  ).toUTCString()